apps/nextjs-website/react-components/components/ServiceCarousel/ServiceCarousel.helpers.tsx

```diff
@@ -63,36 +56,38 @@ export const ServiceCard = (card: ServiceCardProps) => {
       bgcolor={palette.grey[50]}
       borderRadius={4}
       // Styles for mobile image
-      position="relative"
-      overflow="hidden"
+      position='relative'
+      overflow='hidden'
     >
       <Stack
         gap={1}
         width={{ xs: 216, sm: 264, md: 264 }} // 216 has been found by testing: a higher number breaks react-slick and makes the last slide wrap under the first one
         alignSelf={'stretch'}
       >
-        <Typography variant="h6">{card.title}</Typography>
+        <Typography variant='h6' sx={{ fontWeight: '700' }}>
```

Comment:
```suggestion
        <Typography variant='h6' fontWeight={700}>
```
Avoid `sx` if at all possible.